core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterDMTest.java

```diff
@@ -11630,4 +11630,30 @@ private static RelNode toLogical(RelNode rel, RelBuilder builder) {
     assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedBigQuerySql));
   }
 
+  @Test public void testFloorOracle() {
```

Comment:
it should be testFloorOracle or testFloorSpark? 
operators were defined for spark  it seems 
if its correct, then change for the next test case as well

Reply:
Changed the test case name as suggested